translations/messages.fr.yml

```
@@ -410,6 +410,32 @@ pages:
         list:
             title: "Activités"
 
+search:
+    title: "Résultats de recherche"
+    form:
+        placeholder: "Rechercher"
+        filters: "Rechercher"
+        result_types:
+            title: "Type de résultat"
+            pages: "Pages"
+            articles: "Actualités"
+            discussions: "Discussions"
+            documents: "Documents"
+            members: "Membres"
+        groups:
+            title: "Groupes"
+            all_groups: "Tous les groupes"
+            my_groups: "Groupes"
+        particular_groups: "Rechercher dans un groupe précis"
+        tag_list: "Recherche"
+    results:
+        results: "résultats"
+        page: "PAGE"
+        articles: "ACTUALITE"
+        discussion: "DISCUSSION"
+        document: "DOCUMENT"
+        membre: "MEMBRE"
```

```suggestion
        page: "Page"
        articles: "Actualité"
        discussion: "Discussion"
        document: "Document"
        membre: "Membre"
```

C'est mieux d'utiliser ensuite CSS pour les passer en majuscules (avec `text-transform: uppercase`)

---

Ok je  change ça